test/compilable/test21217.d

```diff
@@ -0,0 +1,58 @@
+/*
+REQUIRED_ARGS: -HC=verbose -c -o-
+PERMUTE_ARGS:
+TEST_OUTPUT:
+---
+// Automatically generated by Digital Mars D Compiler v$n$
```

Comment:
Also this is rather annoying because it breaks AUTO_UPDATE.
Do we really need to include the version number in the header message?

Reply:
Might be useful for bug reports but it could also be omitted for `-HC=silent`

Reply:
-> https://github.com/dlang/dmd/pull/11680